test/server/cards/02_SHD/units/WreckerBoom.spec.ts

```diff
@@ -0,0 +1,55 @@
+describe('Wrecker, Boom!', function() {
+    integration(function(contextRef) {
+        describe('Wrecker\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['wrecker#boom'],
+                        leader: 'sabine-wren#galvanized-revolutionary',
+                        groundArena: ['greedo#slow-on-the-draw'],
+                        resources: ['wampa', 'battlefield-marine', 'wild-rancor', 'protector', 'devotion', 'restored-arc170']
+                    },
+                    player2: {
+                        groundArena: ['atst'],
+                        spaceArena: ['green-squadron-awing']
+                    }
+                });
+            });
+
+            it('should defeat a resource and deal 5 damage to a ground unit', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.wrecker);
+
+                // select a resource to defeat
+                expect(context.player1).toBeAbleToSelectExactly([context.wampa, context.battlefieldMarine, context.wildRancor, context.protector, context.devotion, context.restoredArc170]);
+                expect(context.player1).toHavePassAbilityButton();
+                expect(context.player1).toHaveChooseNoTargetButton();
+                context.player1.clickCard(context.devotion);
```

Comment:
Can we also do a resource count after this? Or something else that confirms that the resource has been destroyed?